### ISSUE TITLE
Unknown entries in audit log when using TCP application access

### ISSUE THREAD

**Issue by webvictim** [gravitational/teleport#15397]
Expected behavior: Audit log entries should render correctly.

Current behavior: Audit log entries show up as "Unknown"

![Screenshot 2022-08-10 at 16 39 59](https://user-images.githubusercontent.com/897821/184005679-b1123ff6-be0f-41c0-b93b-8d92fe9a22a7.png)

![Screenshot 2022-08-10 at 16 40 07](https://user-images.githubusercontent.com/897821/184006161-6f77b629-330b-4425-8acd-c008eefb8c0f.png)

Recreation steps:
- Add an application with TCP forwarding
```yaml
app_service:
  enabled: yes
  apps:
  - name: "rdp-example"
    description: "RDP port on example.teleportdemo.com"
    uri: tcp://example.teleportdemo.com:3389
    labels:
      service: rdp
```
- `tsh app login <app>`
- `tsh proxy app <app>`
- Connect to app, disconnect, then view audit log

Bug details:
- Teleport version: `Teleport v10.1.2 git:v10.1.2-0-g5bc7caf go1.18.3`

**Comment by zmb3** [comment]
Hey, at least @xacrimon's automatic handling of unknown events is working well! 😀

**Comment by r0mant** [comment]
This is fixed now, will make it into next 10.0 path release.
